### PR TITLE
[SK-45 Upsert support]

### PR DESCRIPTION
**Pull request by skyflow-puneet**

### CODE REVIEW

File: Samples/CollectAndRevealSample/CollectAndRevealSample/ViewController.swift [hunk 1]
```diff
@@ -47,7 +47,7 @@ class ViewController: UIViewController {
         let tokenProvider = ExampleTokenProvider()
         
         
-        let config = Skyflow.Configuration(vaultID: "<VAULT_ID>", vaultURL: "<VAULT_URL>", tokenProvider: tokenProvider)
+        let config = Skyflow.Configuration(vaultID: "d71fa3bd49c44a74afd3477c6e197615", vaultURL: "https://sb.area51.vault.skyflowapis.dev", tokenProvider: tokenProvider)
```
Comment (yennamsanthosh-skyflow) [inline Samples/CollectAndRevealSample/CollectAndRevealSample/ViewController.swift:50]: Please remove vault id and vault url

File: Samples/UpsertFeature/UpsertFeature/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+//  Created by Tejesh Reddy Allampati on 16/09/21.
+//
+
+
+import UIKit
+import Skyflow
+
+
+
+class ViewController: UIViewController {
```
Comment (yennamsanthosh-skyflow) [inline Samples/UpsertFeature/UpsertFeature/ViewController.swift:18]: samples/UpsertFeature and samples/Upsert seems to be same. Please remove any one of them

File: Samples/CollectAndRevealSample/CollectAndRevealSample/ViewController.swift (hunk 1 above)
Comment (skyflow-puneet) [reply Samples/CollectAndRevealSample/CollectAndRevealSample/ViewController.swift]: done

File: Samples/UpsertFeature/UpsertFeature/ViewController.swift (hunk 2 above)
Comment (skyflow-puneet) [reply Samples/UpsertFeature/UpsertFeature/ViewController.swift:18]: removed Upsert folder

**Pull request merged**